process/incubation.md

```diff
@@ -105,6 +105,9 @@ Changes to the Swift Server Ecosystem index page will be announced by the SSWG u
   * Follow Swift Code style guidelines
   * Uses force unwraps and force tries only as preconditions, ie. conditions that the programmer regards as impossible or programmer error. All force tries/unwraps should come with a comment stating the reasons
   * Prefer native Swift over C wrapping, where appropriate
+* Does not use `*Unsafe*` unless interfacing with C.
```

Comment:
Oops. Looks like the indentation was messed up from my first suggestion

```suggestion
  * Does not use `*Unsafe*` unless interfacing with C
    * Exceptions to uses of `*Unsafe*` constructs are tolerated if appropriately documented why they are absolutely necessary
    * When `*Unsafe*` is used in this manner, it would be appreciated for feature requests to be made for Swift or SwiftNIO in order to make the use of the `*Unsafe*` construct obsolete
```